packages/flame_bloc/DEPRECATED_README.md

```diff
@@ -0,0 +1,55 @@
+# Flame Bloc 🔥🧱
```

Comment:
Perhaps add a comment here about when this deprecated readme can be removed. This way we make sure the file won't linger forever.